reference/enterprise-api-ref/inventory.markdown

```diff
@@ -51,6 +51,9 @@ Inventory API allows to access inventory reports and attributes dictionary.
     | not_match   |
     | contains    |
     | not_contain |
+    | [regex_matches](https://en.wikibooks.org/wiki/Regular_Expressions/POSIX-Extended_Regular_Expressions) |
```

Comment:
I'm not sure I would include a link. I think regex is easy enough to look up or just "know". :)

Reply:
Ok, I will remove it.

Reply:
@craigcomstock done